app/views/content/non-uk-teachers/get-an-international-relocation-payment.md

```diff
@@ -188,9 +188,9 @@ You will not be eligible for the international relocation payment if you are:
 
 ## How teachers can apply for the international relocation payment
 
-Applications for the [international relocation payment (IRP)](https://get-a-teacher-relocation-payment.education.gov.uk) for teachers are open from 2 January 2024 to 29 February 2024. 
+Applications for the [international relocation payment (IRP)](https://get-a-teacher-relocation-payment.education.gov.uk) for teachers are are now closed. They will reopen between 2 April and 31 May 2024. 
 
-To remain eligible for the IRP, you must apply in either the first or second term of your employment as a teacher. Applications will reopen between May and June 2024.
+To remain eligible for the IRP, you must apply in either the first or second term of your employment as a teacher. 
 
 You will need a teaching job to apply for the IRP. [Learn more about finding a job and applying for your visa](/non-uk-teachers/teach-in-england-if-you-trained-overseas).
```

Comment:
@laurahermionetennant92 this sentence here confused me. It says you need to apply in either your 1st or 2nd term of employment as a teacher. Then it says you need a teaching job to apply.

Might be better to switch this round? and say: You'll need a teaching job to apply for the IRP. And you need to apply for the IRP in either the first or second term of your employment to be eligible for it.

Or slightly better worded thatn that!

Reply:
See what you mean, have swapped